pyccel/stdlib/lists/lists.h

```diff
@@ -0,0 +1,74 @@
+
+#ifndef _LISTS_
+#define _LISTS_
+#define DEFAULT_CAP 10
+
+#include <stdio.h>
+#include <stddef.h>
+#include <string.h>
+#include <stdlib.h>
+#include <complex.h>
+
+#define GET_INDEX(list, i) (list->elements + (i * tSizes[list->type]))
```

Comment:
Are you sure this function returns the index? Not the address at that index?

Reply:
If it is supposed to return the element, why do you do:
```
list->elements + (i * tSizes[list->type])
```
rather than
```
list->elements[i * tSizes[list->type]]
```
?

Reply:
https://github.com/pyccel/pyccel/pull/1192/commits/67142a65365551d27535ef09b33dc0d3a7717e69

Reply:
> If it is supposed to return the element, why do you do:
> 
> ```
> list->elements + (i * tSizes[list->type])
> ```
> 
> rather than
> 
> ```
> list->elements[i * tSizes[list->type]]
> ```
> 
> ?

it's supposed to give the address of an element from the list, and then you can cast it to what ever type you want.